src/main/java/seedu/duke/Duke.java

```diff
@@ -23,9 +26,15 @@ public static void main(String[] args) {
                 "          | |       | |                                                          __/ |          \n" +
                 "          |_|       |_|                                                         |___/";
         System.out.println("Hello from\n" + logo);
+        Duke duke = new Duke();
+        try {
+            storage.loadData(equipmentInventory);
+        } catch (DuplicateSerialNumber e) {
```

Comment:
Can we catch the DuplicateSerialNumberException in the storage.loadData() method instead? Then I think we can simply skip the line of data with duplicate serial number and continue with adding the rest.